setup.php

```diff
@@ -1349,147 +1410,168 @@ function cloneClicked() {
+                        echo "<div class='jumbotron p-5'>";
                         echo "<p>Configuration of Apache web server...</p><br />\n";
-                        echo "The <strong>\"" . preg_replace("/${site_id}/", "*", realpath($docsDirectory)) . "\"</strong> directory contain patient information, and
+                        echo "The <code>\"" . preg_replace("/${site_id}/", "*", realpath($docsDirectory)) . "\"</code> directory contain patient information, and
                         it is important to secure these directories. Additionally, some settings are required for the Zend Framework to work in OpenEMR. This can be done by pasting the below to end of your apache configuration file:<br /><br />
-                        &nbsp;&nbsp;&lt;Directory \"" . realpath(dirname(__FILE__)) . "\"&gt;<br />
+                        &nbsp;&nbsp;<code>&lt;Directory \"" . realpath(dirname(__FILE__)) . "\"&gt;<br />
                         &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;AllowOverride FileInfo<br />
                         &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Require all granted<br />
-                        &nbsp;&nbsp;&lt;/Directory&gt;<br />
+                        &nbsp;&nbsp;<code>&lt;/Directory&gt;</code><br />
                         &nbsp;&nbsp;&lt;Directory \"" . realpath(dirname(__FILE__)) . "/sites\"&gt;<br />
                         &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;AllowOverride None<br />
-                        &nbsp;&nbsp;&lt;/Directory&gt;<br />
-                        &nbsp;&nbsp;&lt;Directory \"" . preg_replace("/${site_id}/", "*", realpath($docsDirectory)) . "\"&gt;<br />
+                        &nbsp;&nbsp;&lt;/Directory&gt;</code><br />
+                        &nbsp;&nbsp;<code>&lt;Directory \"" . preg_replace("/${site_id}/", "*", realpath($docsDirectory)) . "\"&gt;<br />
                         &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Require all denied<br />
-                        &nbsp;&nbsp;&lt;/Directory&gt;<br /><br />";
+                        &nbsp;&nbsp;&lt;/Directory&gt;</code><br /><br />";
 
                         $btn_text = 'Proceed to Select a Theme';
                         $step5_bottom = <<<STP5BOT
-                        <p>If you are having difficulty finding your apache configuration file, then refer to the <a href='Documentation/INSTALL' rel='noopener' target='_blank'><span style='text-decoration: underline;'>'INSTALL'</span></a> manual for suggestions.</p>
+                        <p>If you are having difficulty finding your apache configuration file, then refer to the <a href='Documentation/INSTALL' rel='noopener' target='_blank'><u>'INSTALL'</u></a> manual for suggestions.</p>
                         <p>We recommend you print these instructions for future reference.</p>
                         <p class='mark'>Click <strong>'$btn_text'</strong> to select a theme.</p>
                         <br />
                         <form method='post'>
```

Comment:
Hi @tywrenn 

Should I replace the `.mark` class to HTML mark tag?

Reply:
I would leave it alone due to differing styles between the p tag and the mark tag